stories/Organism/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { FaqFun, SidebarMenu } from '../../assets/js/Faq_custom'
+import './sidebar.scss';
+
+export const Sidebar = ({ headerText, data, label }) => {
+  useEffect(() => {
+    FaqFun();
+    SidebarMenu();
+  }, [])
+
+  return (
+    <div className='grid-x grid-margin-x'>
+      <div className='cell medium-4'>
+        <ul className='accordion-navbar' id="accordion">
+          <span className='accordion-menu hide-for-large'>{label}</span>
```

Comment:
@pallavikadam-srijan We can't have the span directly in ul and it should be wrapped in li

Reply:
@surbhi90 Done